README.md

```diff
@@ -25,3 +25,6 @@ Unless you explicitly state otherwise, any contribution intentionally
 submitted for inclusion in the work by you, as defined in the Apache-2.0
 license, shall be dual licensed as above, without any additional terms
 or conditions.
+
+[docs]: https://vorner.github.io/spirit/doc/spirit/index.html
```

Comment:
Will this get published together with the build as well?

Reply:
That's what I had in mind when doing https://github.com/vorner/spirit/pull/43/commits/69abaa62bb9964225991881791de353f7f0b0d31. The idea is that the user guide will be able to link to the current (`master`) API docs instead of a particular version on docs.rs.